src/Microsoft.Docs.Template/resources/tokens.ar-SA.resx

```diff
@@ -0,0 +1,135 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+  <!-- 
```

Comment:
nit: can we get rid of these large section of comments?

Reply:
its' a default schema template, which can help to integrate the resx with vs